icu4c/source/test/intltest/intltest.cpp

```diff
@@ -2039,6 +2039,29 @@ UBool IntlTest::assertEquals(const char* message,
     return TRUE;
 }
 
+UBool IntlTest::assertEquals(const char* message,
+                             double expected,
+                             double actual,
+                             double delta) {
+    if (std::isnan(delta) || std::isinf(delta)) {
+        errln((UnicodeString)("FAIL: ") + message + "; nonsensical delta " + delta +
+              " - delta may not be NaN or Inf");
+        return FALSE;
+    }
+    bool bothNaN = std::isnan(expected) && std::isnan(actual);
+    double difference = abs(expected - actual);
```

Comment:
Absolute difference is good here.

Reply:
(Taking this as just an observation, not a suggested change.)